CodeStarterCamp_Week3/CoffeeShop.swift

```diff
@@ -7,34 +7,53 @@
 
 import Foundation
 
-struct CoffeeShop {
+class CoffeeShop {
     var name: String
     var sales: Int
     var barista: Person
     var menu: [Coffee: Int]
-    var pickUpTable: [Coffee.RawValue]
+    var pickUpTable: [Coffee] {
+        didSet {
+            coffeeOnPickUpTable()
+        }
+    }
     
-    init(name: String, sales: Int, barista: Person, menu: [Coffee : Int], pickUpTable: [Coffee.RawValue]) {
+    init(name: String, sales: Int, barista: Person, menu: [Coffee : Int], pickUpTable: [Coffee]) {
         self.name = name
         self.sales = sales
         self.barista = barista
         self.menu = menu
         self.pickUpTable = pickUpTable
     }
     
-    mutating func takeOrderCoffee(coffeeMenu:Coffee) {
+    func takeOrderCoffee(_ coffeeMenu:Coffee) {
         if let price = menu[coffeeMenu] {
-            print("\(coffeeMenu)주문 받았습니다.")
             print("주문하신 \(coffeeMenu)의 가격은\(price)입니다.")
             sales += price
-            pickUpTable.append(coffeeMenu.rawValue)
+            pickUpTable.append(coffeeMenu)
+            print("매출액 \(sales)원 입니다.")
         } else {
             print("주문 하신 \(coffeeMenu)은 저희 매장에 없는 메뉴 입니다.")
         }
     }
     
+    func make(_ coffee: Coffee, for naem: String) {
+        if let price = menu[coffee] {
+            sales += price
+            pickUpTable.append(coffee)
+            print("커피의 가격은 \(price)입니다.")
+            print("매출액 \(sales)원 입니다.")
```

Comment:
```suggestion
            print("커피의 가격은 \(price)입니다.")
            sales += price
            print("매출액 \(sales)원 입니다.")
            pickUpTable.append(coffee)
```

금액에 대한 안내를 하고, 돈을 받고, 매출액을 파악하는 흐름이 자연스러울 것 같습니다!